client/webserver/site/src/js/wallets.js

```diff
@@ -150,6 +147,21 @@ export default class WalletsPage extends BasePage {
     }
   }
 
+  /*
+   * setPWSettingViz sets the visibility of the password field section.
+   */
+  setPWSettingViz (visible) {
+    if (visible) {
+      Doc.hide(this.page.showIcon)
+      Doc.show(this.page.hideIcon, this.page.changePW)
+      this.page.switchPWMsg.textContent = 'exclude password field'
+      return
+    }
+    Doc.hide(this.page.hideIcon, this.page.changePW)
+    Doc.show(this.page.showIcon)
+    this.page.switchPWMsg.textContent = 'include password field'
```

Comment:
Forgot to mention that I find "include password field" and "exclude password field" very confusing as they hint at implementation and not meaning to the user.

How about "+ set a new wallet password" and "- do not set a new wallet password"?